src/shell.c

```diff
@@ -52,13 +52,13 @@ int main() {
     char hostname[MAX_SIZE];
     gethostname(hostname, sizeof(hostname));
 
-    getcwd(cwd, sizeof(cwd));
-    printf("%s@%s:%s$ ", username, hostname, cwd);
     while (1) {
+        getcwd(cwd, sizeof(cwd));
```

Comment:
Nu ai nevoie de linia asta, pune direct in printf getcwd(NULL, MAX_SIZE) ca sa nu mai folosesti spatiu si pentru cwd.